workshop/config.toml

```diff
@@ -9,7 +9,7 @@ metaDataFormat = "yaml"
 defaultContentLanguageInSubdir= true
 
 uglyurls = false
-googleAnalytics = "UA-151248716-1"
+#googleAnalytics = "UA-151248716-1"
```

Comment:
why not just delete rather than comment? it's in the git history anyways

Reply:
shouldn't the script be removed from the template as well?

Reply:
> shouldn't the script be removed from the template as well?

It's [built-in functionality](https://gohugo.io/templates/internal/) for Hugo.  I don't think we need to fork it to remove the unused templates.

Reply:
Thanks Nick, line has been removed.

Reply:
@toksvaeth footer has been updated.